site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+```
+
+Refresh the browser and the zombie animation should now play / pause when clicked.
+
+{% section %}
+
+## Animation events
+
+The webanimation meta provides callback functions for `onFinish` and `onCancel`. To ensure that we can replay our animation we need to set `this._play` to false when the animation has finished, to do this we will add an `onFinish` callback.
+
+{% instruction 'Add onFinish callback to Zombies.ts' %}
+
+``` typescript
+// Add the onFinish function
+private _onAnimationFinish() {
+	this._play = false;
+	this.invalidate();
+}
+
+//add to the top of the render function
+const zombieOneMoveAnimation = {
+	id: 'zombieOneMove,
+	effects: [
+		{ left: '0%' },
+		{ left: '35%' }
+	],
+	timing: {
+		duration: 8000,
+		easing: 'ease-in',
+		fill: 'forwards'
+	},
+	controls: {
+		play: true,
+		onFinish: this._onAnimationFinish // Add the callback here
+	}
+};
+```
+
+We will use this `onFinish` callback later in the tutorial to trigger a second animation.
+
+## Lets add more animations
+
+Not content with your zombie just sliding across the screen like that? Let's add some hollywood style move effects by animating the legs and the body.
+
+{% instruction 'Add more animations to Zombies.ts' %}
+
+``` typescript
+private _getZombieBodyAnimation(id: string): AnimationProperties {
+		return {
+			id,
+			effects: [
+				{ transform: 'rotate(0deg)' },
+				{ transform: 'rotate(-2deg)' },
+				{ transform: 'rotate(0deg)' },
+				{ transform: 'rotate(3deg)' },
+				{ transform: 'rotate(0deg)' }
+			],
+			timing: {
+				duration: 1000,
+				iterations: Infinity
+			},
+			controls: {
+				play: this._play
+			}
+		};
+	};
+
+	private _getZombieLegAnimation(id: string, front?: boolean): AnimationProperties {
+		const effects = [
+			{ transform: 'rotate(0deg)' },
+			{ transform: 'rotate(-5deg)' },
+			{ transform: 'rotate(0deg)' },
+			{ transform: 'rotate(5deg)' },
+			{ transform: 'rotate(0deg)' }
+		];
+
+		if (front) {
+			effects.reverse();
+		}
+
+		return {
+			id,
+			effects,
+			timing: {
+				duration: 1000,
+				iterations: Infinity
+			},
+			controls: {
+				play: this._play
+			}
+		};
+	}
+
+	// Add this to the render function with the other `animate` calls
+	webAnimation.animate('zombieOneBody', this._getZombieBodyAnimation('zombieOneBody'));
+	webAnimation.animate('zombieOneBackLeg', this._getZombieLegAnimation('zombieOneBackLeg'));
+	webAnimation.animate('zombieOneFrontLeg', this._getZombieLegAnimation('zombieOneFrontLeg', true));
+	webAnimation.animate('zombieTwoBody', this._getZombieBodyAnimation('zombieTwoBody'));
+	webAnimation.animate('zombieTwoBackLeg', this._getZombieLegAnimation('zombieTwoBackLeg'));
+	webAnimation.animate('zombieTwoFrontLeg', this._getZombieLegAnimation('zombieTwoFrontLeg', true));
+```
+
+Refresh your browser and you should now see two stumbling zombies moving towards one another. Feel free to have a play with the params for rotation / timings to get it just how you like it.
```

Comment:
"to have a play with the params for rotation / timings to get it just how you like it" -> to experiment with the parameters for rotation and timings